src/catch2/matchers/catch_matchers_all.hpp

```diff
@@ -25,6 +25,7 @@
 #include <catch2/matchers/catch_matchers_contains.hpp>
 #include <catch2/matchers/catch_matchers_exception.hpp>
 #include <catch2/matchers/catch_matchers_floating.hpp>
+#include <catch2/matchers/catch_matchers_generic.hpp>
```

Comment:
This is not a good filename given the contents. "Logical" would be better, but it still doesn't feel ideal.

Reply:
Do you mean I need to create a file for each matcher? e. g. `catch_matchers_addall.hpp`

Reply:
Not necessarily. But I don't think people would expect to find `AllMatch`, `AnyMatch`, `NoneMatch` in file named "matchers generic", when the documentation already talks about "Generic Matchers" as a way of defining new matchers. Ideally there would be a good name for them as a group, but I can't think of any.

Placing them each into a separate file works too I guess, e.g. `catch_matchers_allmatch.hpp`

Reply:
What about quantifiers? `catch_matchers_quantifiers.hpp` which includes `AllMatch`, `AnyMatch` and `NoneMatch`.

Reply:
I won't say that I think it is great, but since I've already gotten "quantifiers" as a suggestion from someone else, it is apparently not as unintuitive as I though.

So, sure, go for it. 👍

Reply:
Great, I'll update it!